lib/peeps.rb

```diff
@@ -0,0 +1,57 @@
+require 'pg'
+
+class Peeps
+
+  attr_reader :id, :message, :entry_date, :filter
+  
+  def initialize(id:, message:, entry_date:)
+    @id = id
+    @message = message
+    @entry_date = entry_date
+  end
+
+  def self.open_connection
```

Comment:
It's possible to refactor this into its own class and have an always on db connection while the app is running. There's a walkthrough on how to do this in Ex.15 on the Bookmark task.